lib/harvest_utils.rb

```diff
@@ -310,6 +311,13 @@ def self.normalize_facets(doc, string_to_search)
       end
     end
 
+    def self.normalize_dates(doc, string_to_search)
+      node_update = doc.search(string_to_search)
+      node_update.each do |node_value|
+        node_value.inner_html = node_value.inner_html.gsub(/[circacaCIRCACA.]/, '').strip
```

Comment:
`gsub(/[circacaCIRCACA.]/, '')` will delete all instances of the letters listed within brackets. Rather, `gsub(/(circa|ca|CIRCA|CA.)/, '')` will delete instances of the various forms of the term circa.